plugin/src/main/scala/Plugin.scala

```diff
@@ -22,19 +28,31 @@ object DatatypePlugin extends AutoPlugin {
     }
 
     lazy val baseDatatypeSettings: Seq[Def.Setting[_]] = Seq(
+      createDatatypes in generateDatatypes := true,
+      createJsonFormats in generateDatatypes := true,
       datatypeJavaLazy in generateDatatypes := "xsbti.api.Lazy",
       datatypeSource in generateDatatypes := Defaults.configSrcSub(sourceDirectory).value / "datatype",
       sourceManaged in generateDatatypes := sourceManaged.value,
       datatypeScalaFileNames in generateDatatypes := scalaDef2File,
       // We cannot enable this by default, because the default function for naming Scala files that we provide
       // will create a separate file for every `Definition`.
       datatypeScalaSealProtocols in generateDatatypes := false,
+      datatypeSerializerPackage in generateDatatypes := Some("serialization"),
+      datatypeSerializerName in generateDatatypes := "Serializer",
+      datatypeSerializerParents in generateDatatypes := Nil,
+      datatypeInstantiateJavaLazy in generateDatatypes := { (e: String) => s"xsbti.SafeLazy($e)" },
```

Comment:
Do you mean that sbt-datatype should provide a library that would define concrete instances of `xsbti.api.Lazy[T]`? At the moment there's a `settingKey` to define the name of the lazy type to use.